src/client/deserializers/cluster.ts

```diff
@@ -0,0 +1,21 @@
+/*
+ * Copyright 2017-2022 Allegro.pl
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { ClientCluster, SerializedCluster } from "../../common/models/cluster/cluster";
+
+export function deserialize(cluster: SerializedCluster): ClientCluster {
+  return cluster;
```

Comment:
So this is just for consistency - all models have serializers and deserializers?

Reply:
More important is that every model has client and serialised version. Something that lives in browser and something that can be transferred via wire. Here by happy coincidence, client object is build from JSON primitives so it can be easily deserialised. But some time in the future we may have non primitive fields and deserialisation will be more complicated than identity function.